sdk/purview/purview-workflow-rest/README.md

```diff
@@ -0,0 +1,140 @@
+# Azure Purview Workflow Rest-Level client library for JavaScript
+
+Workflows are automated, repeatable business processes that users can create within Microsoft Purview to validate and orchestrate CUD (create, update, delete) operations on their data entities. Enabling these processes allow organizations to track changes, enforce policy compliance, and ensure quality data across their data landscape.
+
+Use the client library for Purview Workflow to:
+
+- Manage workflows
+- Submit user requests and monitor workflow runs
+- View and respond to workflow tasks
+
+**For more details about how to use workflow, please refer to the [service documentation][product_documentation]**
+
+## Getting started
+
+### Currently supported environments
+
+- Node.js version 14.x.x or higher
+
+### Prerequisites
+
+- You must have an [Azure subscription][azure_subscription] and a [Purview resource][purview_resource] to use this package.
+
+### Create and authenticate a `PurviewWorkflowClient`
+
+Since the Workflow service uses an Azure Active Directory (AAD) bearer token for authentication and identification, an email address should be encoded into the token to allow for notification when using Workflow. It is recommended that the [Azure Identity][azure_identity] library be used  with a the [UsernamePasswordCredential][username_password_credential]. Before using the [Azure Identity][azure_identity] library with Workflow, [an application][app_registration] should be registered and used for the clientId passed to the [UsernamePasswordCredential][username_password_credential].
+Set the values of the client ID, tenant ID, username and password as environment variables:
+AZURE_CLIENT_ID, AZURE_TENANT_ID, USERNAME, PASSWORD
+
+```typescript
+import PurviewWorkflow from "@azure-rest/purview-workflow";
+import { UsernamePasswordCredential } from "@azure/identity";
+const client = PurviewWorkflow(
+  "https://<account-name>.purview.azure.com",
+  new UsernamePasswordCredential(
+        <AZURE_TENANT_ID>,
+        <AZURE_CLIENT_ID>,
+        <USERNAME>,
+        <PASSWORD>
+      )
+  );
+```
+
+## Examples
+
+The following section provides several code snippets covering some of the most common scenarios, including:
+
+- [Submit User Requests](#submit-user-requests)
+- [Approve Workflow Task](#approve-workflow-task)
+
+### Submit user requests
+
+```typescript
+import createPurviewWorkflowClient, {
+  SubmitUserRequestsParameters
+} from "@azure-rest/purview-workflow";
+import { UsernamePasswordCredential } from "@azure/identity";
+
+async function userRequestsSubmit() {
+  const endpoint = <ENDPOINT>;
+  const tenantId = <TENANTID>;
+  const clientId = <CLIENTID>;
+  const username = <USERNAME>;
+  const password = <PASSWORD>;
```

Comment:
We should encourage our users to read these from the environment variables to avoid having them leak their secrets by mistake. Please take a look at other samples in this repo to see how we are doing it. We read them from `process.env`

Reply:
Fixed, thanks